lib/app.js

```diff
@@ -1,17 +1,24 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
-
-const authenticate = require('./middleware/authenticate');
-
+const cors = require('cors');
 const app = express();
 
+// const authenticate = require('./middleware/authenticate');
+
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use(
+  cors({
+    origin: ['http://localhost:5500'],
+    credentials: true,
+  })
+);
 
 // App routes
 
 app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/items', require('./controllers/items.js'));
```

Comment:
Since everything on this route requires the authenticate middleware you can put it at the top level here and then you wouldn't have to type it in every route in the controller.